.github/workflows/ghcr-build.yml

```diff
@@ -85,14 +86,35 @@ jobs:
         run: |
           ./containers/build.sh -i openhands -o ${{ env.REPO_OWNER }} --push
 
+      - name: Run Trivy vulnerability scanner
+        if: github.event_name != 'pull_request'
```

Comment:
Starting by scanning non-PRs only so we don't add a few minutes to the docker build. Had to comment out those conditionals to test it which isn't great. Alternatives are add the time, run on a subset of PRs, or do some complicated cache juggling to reuse the cache from main branch.